blog/docs/releases/announcing-knative-v1-4-release.md

```diff
@@ -0,0 +1,209 @@
+---
+title: "v1.4 release"
+linkTitle: "v1.4 release"
+Author: "[Nader Ziada](https://twitter.com/nziada)"
+Author handle: https://github.com/nader-ziada
+date: 2022-04-20
+description: "Knative v1.4 release announcement"
+type: "blog"
+---
+
+
+## Announcing Knative 1.4 Release
+
+A new version of Knative is now available across multiple components.
+
+Follow the instructions in
+[Installing Knative](https://knative.dev/docs/install/) to install the components you require.
+
+## Table of Contents
+- [Highlights](#highlights)
+- [Serving v1.4](#serving-v1.4-NOSPACES)
+- [Eventing v1.4](#eventing-v1.4-NOSPACES)
+- [Eventing Extensions](#eventing-extensions)
+    - [Apache Kafka Broker v1.4](#apache-kafka-broker-v1.4-NOSPACES)
+    - [RabbitMQ Broker and Source v1.4](#rabbitmq-broker-and-source-v1.4-NOSPACES)
+- `kn` [CLI v1.4](#client-v1.4-NOSPACES)
+- [Knative Operator v1.4](#operator-v1.4-NOSPACES)
+- [Thank you contributors](#thank-you-contributors)
+
+
+## Highlights
+
+- Our minimum supported Kubernetes version is now 1.22
+
+
+## Serving v1.4
+
+<!-- Original notes are here: https://github.com/knative/serving/releases/tag/knative-v1.4.0 -->
+
+🚨 Breaking or Notable
+ - Our minimum supported Kubernetes version is now 1.22 (#12753, @dprotaso)
+ - When using the Horizontal Pod Autoscaler (HPA), revisions will only be marked as ready after the initial-scale / min-scale value is reached. For example, if min-scale: "4", the revision will not be marked ready until all four pods are ready. Note that revisions may take slightly longer to become ready after this change. See https://knative.dev/docs/serving/autoscaling/scale-bounds/ for more details. (#12811, @psschwei)
+
+💫 New Features & Changes
+ - Support annotations 'networking.knative.dev/http-protocol' for Overriding the default HTTP behavior per service in DomainMapping CRD. (#12786, @wei840222)
+ - serving.knative.dev/release labels, deprecated in v1.3, have been removed. Please switch over to using app.kubernetes.io/name: knative-serving and app.kubernetes.io/version: $VERSION. (#12754, @psschwei)
+ - Users can configure a per-revision progress deadline by setting the serving.knative.dev/progress-deadline annotation in .spec.template.metadata.annotations (#12751, @psschwei)
+
+🐞 Bug Fixes
+ - Changes the default target-burst-capacity to 210 in order to fix a configuration issue that caused rapid swapping of activator in/out of path. (#12774, @psschwei)
+ - Fix gc: delete revision in correct order (#12752, @Taction)
+ - Fix tag to digest resolution when using imagePullSecrets (#12836, @dprotaso)
+
+## Eventing v1.4
+
+<!-- Original notes are here: https://github.com/knative/eventing/releases/tag/knative-v1.4.0 -->
+
+### 🚨 Breaking or Notable Changes
+
+- The minimum Kubernetes version is 1.22. (#6280, @pierDipi)
+
+### 💫 New Features & Changes
+
+- The sugar reconciler has been integrated into the base eventing controller and is now controlled by two LabelSelector fields.
```

Comment:
Can this be under a `🚨 Breaking or Notable Changes` section as its modifying existing functionality?

Reply:
done